test/clt-tests/sharding/syntax/queries-syntax-quotes.rec

```diff
@@ -0,0 +1,106 @@
+––– block: ../base/init –––
+––– input –––
+export INSTANCE=1
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+mysql -h0 -P1306 -e "create table u(id bigint) shards=10 rf=1"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "SHOW TABLES\G"
```

Comment:
It's better to validate that we have required table created and it's distributed, due to future changes and other tables probably will be hidden from output

Reply:
Maybe that's what you mean. 
```
––– input –––
mysql -h0 -P1306 -e "DESCRIBE u;"
––– output –––
+-------+-------+
| Agent | Type  |
+-------+-------+
| u_s0  | local |
| u_s1  | local |
| u_s2  | local |
| u_s3  | local |
| u_s4  | local |
| u_s5  | local |
| u_s6  | local |
| u_s7  | local |
| u_s8  | local |
| u_s9  | local |
+-------+-------+
––– input –––
mysql -h0 -P1306 -e "DESCRIBE b;"
––– output –––
+-------+--------+------------+
| Field | Type   | Properties |
+-------+--------+------------+
| id    | bigint |            |
+-------+--------+------------+
```

Reply:
All tables like u_s0...10 and so on will be hidden tables and not displayed by default, so it seems to be good idea just to validate that we have only requested table created, all others are internals